libs/@guardian/eslint-config/project.json

```diff
@@ -10,10 +10,17 @@
 				"outputPath": "dist/libs/@guardian/eslint-config",
 				"packageJson": "libs/@guardian/eslint-config/package.json",
 				"assets": [
-					"libs/@guardian/eslint-config/.eslintrc.js",
+					"libs/@guardian/eslint-config/index.js",
```

Comment:
exactly sorry - when @akash1810 and @AshCorr we're debugging the initial problem, they struggled to find the actual config in `node_modules`, hopefully this will address that (also standardises the package structures)